e2e/test-runner/test_utils.go

```diff
@@ -65,14 +65,14 @@ func subiquityLogs(t *testing.T) string {
 
 	ctx, cancel = context.WithTimeout(context.Background(), 30*time.Second)
 	defer cancel()
-	out, err = wslCommand(ctx, "cat", ServerLogPath).CombinedOutput()
+	out, err = wslCommand(ctx, "cat", serverLogPath).CombinedOutput()
 	if err != nil {
 		t.Logf("Failed to retrieve server debug log: %v\n%s", err, out)
 	} else {
 		s = s + fmt.Sprintf("\n==== Server Debug Log ====\n%s\n", out)
 	}
 
-	out, err = ioutil.ReadFile(clientLogFullPath)
```

Comment:
oh sorry, I didn’t spot you were using ioutil during our peer programming :) Be warned, as this change is like a couple of Go release ago, you will find a lot of examples still using ioutil, which is now deprecated (all functionalities have similar helpers in different other packages)

Reply:
Yep, the linter also complained so I fixed it later in this commit: 096600d8c41e3b697c1cf74f3159ff32b6b3fe5d